File: app/src/main/java/com/kroraina/easyreader/modules/rank/BillboardContract.java
package com.kroraina.easyreader.modules.rank;

import com.kroraina.easyreader.base.mvp.BaseContract;



public interface BillboardContract {

    interface View extends BaseContract.BaseView{
        void finishRefresh(BillboardPackage beans);
    }

    interface Presenter extends BaseContract.BasePresenter<View>{
        void loadBillboardList();
    }
}

File: app/src/main/java/com/kroraina/easyreader/modules/rank/BillboardContract.kt
package com.kroraina.easyreader.modules.rank

import com.kroraina.easyreader.base.mvp.BaseContract


interface BillboardContract {

    interface View : BaseContract.BaseView {
        fun finishRefresh(beans: BillboardPackage?)
    }

    interface Presenter : BaseContract.BasePresenter<View> {
        fun loadBillboardList()
    }
}
